packages/react-compose/src/compose.ts

```diff
@@ -23,7 +23,7 @@ function compose<T extends React.ElementType, InputProps, InputStylesProps, Pare
     Component.defaultProps = (input as InputComposeComponent).defaultProps;
   }
 
-  (Component as ComposedComponent).fluentComposeConfig = composeOptions;
+  ((Component as unknown) as ComposedComponent).fluentComposeConfig = composeOptions;
```

Comment:
Just out of curiosity, why the `unknown`?

Reply:
Let me see if I can get rid of it. At some point ts was complaining that ComposedComponent has nothing in common with the initial Components typing, so I had to cast to `unknow`.

Reply:
I tried to remove locally didn't got any errors, maybe it's fixed by now

Reply:
Actually it is ts error, it takes some time for it to process the changes.. I was still hitting this, so I left it as is..